{
	"id": "oval:org.debian:def:206842820377723014218888249124273043939",
	"class": "vulnerability",
	"metadata": {
		"title": "CVE-2022-37454 pypy3",
		"affected": {
			"family": "unix",
			"platform": "Debian GNU/Linux 11",
			"product": "pypy3"
		},
		"reference": [
			{
				"source": "CVE",
				"ref_id": "CVE-2022-37454",
				"ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-37454"
			}
		],
		"description": "security update",
		"debian": {
			"moreinfo": "no info",
			"dsa": "DSA-5269"
		}
	},
	"criteria": {
		"operator": "AND",
		"criterias": [
			{
				"operator": "OR",
				"criterias": [
					{
						"operator": "AND",
						"criterions": [
							{
								"test_ref": "oval:org.debian.oval:tst:2",
								"comment": "all architecture"
							},
							{
								"test_ref": "oval:org.debian.oval:tst:25691",
								"comment": "pypy3 DPKG is earlier than 7.3.5+dfsg-2+deb11u2"
							}
						]
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.debian.oval:tst:1",
				"comment": "Debian 11 is installed"
			}
		]
	}
}
